docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md

```diff
@@ -190,6 +190,10 @@ When you get the external IP you can ssh into the nodes using the private ssh ke
 ssh -i <ssh-private-key> <ssh-username>@<external-IP>
 ```
 
+### "msg"="discovered IP address" log message with an address value of the controlplane in the capv log
```

Comment:
When you say capv log, do you mean the EKS Anywhere log? That part of heading "in the capv log" might not be needed here.